src/screens/Embed/Embed.js

```diff
@@ -20,32 +22,89 @@ import ChartsTab from './ChartsTab';
 import EmbedFooter from './EmbedFooter';
 
 export default function Embed() {
-  const { id: _location } = useParams();
-  const location = _location.toUpperCase();
+  const { id: _location, countyId, countyFipsId } = useParams();
 
-  const [summaryData, setSummaryData] = useState(null);
+  const [stateSummaryData, setSummaryData] = useState(null);
   const [tabState, setTabState] = useState(0);
   const [shareDrawerOpen, setShareDrawerOpen] = useState(false);
   const handleTabChange = (_event, newTabValue) => setTabState(newTabValue);
 
-  const modelDatasMap = useModelDatas(location, null);
+  const [selectedCounty, setSelectedCounty] = useState(null);
+  const [location, setLocation] = useState(null);
+  const [missingCounty, setMissingCounty] = useState(false);
+  useMemo(() => {
+    if (countyId) {
+      const loc = _location.toUpperCase();
+      setLocation(loc);
+      setSelectedCounty(
+        _.find(US_STATE_DATASET.state_county_map_dataset[loc].county_dataset, [
+          'county_url_name',
+          countyId,
+        ]),
+      );
+    } else if (countyFipsId) {
+      const found = findCountyByFips(countyFipsId);
+      setSelectedCounty(found);
+      if (found) {
+        setLocation(found.state_code);
+      } else {
+        setMissingCounty(true);
+      }
+    } else {
+      setLocation(_location.toUpperCase());
+    }
+  }, [_location, countyId, countyFipsId]);
+
+  const modelDatasMap = useModelDatas(location, selectedCounty);
 
   const locationName = STATES[location];
   const intervention = STATE_TO_INTERVENTION[location];
-  const interventions = buildInterventionMap(modelDatasMap.stateDatas);
+
+  const datasForView = selectedCounty
+    ? modelDatasMap.countyDatas
+    : modelDatasMap.stateDatas;
+  let interventions = null;
+  if (datasForView && !datasForView.error) {
+    interventions = buildInterventionMap(datasForView);
+  }
+  let stateInterventions = null;
+  if (modelDatasMap.stateDatas) {
+    stateInterventions = buildInterventionMap(modelDatasMap.stateDatas);
```

Comment:
reminder we are not going to do this and fix the state header by just passing in interventions instead of stateInterventions